### ISSUE TITLE
attention_ocr evaluation on Synth 90k dataset - poor results

### ISSUE THREAD

**Issue by antoniosimunovic** [tensorflow/models#2144]
- **What is the top-level directory of the model you are using**: attention_ocr/python
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: no
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Linux Ubuntu 14.04
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: 1.0.0.

Hi,
when I run evaluation on a subset of Synth 90k dataset (10 examples only), prediction results are very bad (~0.1 character accuracy). I've inspected the predictions and they all seem to reflect labels found in the training FSNS dataset, which is odd. It looks like the model generalization is bad, but that's not what the paper reports, so it must be some error in eval code or my inputs are somehow wrong. I've also ran evaluation on a full Synth 90k test dataset and reported character accuracy is in the same range (~0.1). Can someone make a comment on this?

I'm using pretrained model available at http://download.tensorflow.org/models/attention_ocr_2017_05_17.tar.gz

This is the command:

`python eval.py --split_name test --train_log_dir attention_ocr_2017_05_17 --dataset_name synth90k --num_batches 10
`

These are some of the results: 
- impeaching: Rue du Chauzou
- enticements: Rue le le le le le Tetuint lau...
- slinking: Rue de la Ruere

This is my datasets/synth90k.py file (only relevant lines):

```
DEFAULT_DATASET_DIR = os.path.join(os.path.dirname(file), 'synth90k')
DEFAULT_CONFIG = {
'name':'synth90k',
'splits': {
   'test': {
       'size': 10,
       'pattern': 'synth90k_test*.tfrecord'
   }
},
'charset_filename': 'charset_synth90k.txt',
'image_shape': (31, 200, 3),
'num_of_views':1,
'max_sequence_length': 37,
'null_code': 133,
...
}
```

**Comment by alexgorban** [comment]
I've answered this questions https://stackoverflow.com/a/45552106/743658. Please close the issue.
